frontend/packages/ui/src/components/Button/Button.tsx

```diff
@@ -12,6 +13,8 @@ type Props = ComponentProps<'button'> & {
   size?: 'sm' | 'md' | 'lg'
   leftIcon?: ReactNode
   rightIcon?: ReactNode
+  isLoading?: boolean | undefined
+  loadingIndicatorType?: 'leftIcon' | 'content'
```

Comment:
👍  I understood that "content" here means something like replacing the button content with a spinner icon!